suitcase/spec.py

```diff
@@ -740,13 +739,13 @@ def to_spec_file_header(start, filepath, baseline_descriptor=None):
     return _SPEC_FILE_HEADER_TEMPLATE.render(md)
 
 
-_SPEC_1D_COMMAND_TEMPLATE = env.from_string("{{ plan_type }} {{ scan_motor }} {{ start }} {{ stop }} {{ num }} {{ time }}")
+_SPEC_1D_COMMAND_TEMPLATE = env.from_string("{{ plan_name }} {{ scan_motor }} {{ start }} {{ stop }} {{ num }} {{ time }}")
 
 _SCANS_WITHOUT_MOTORS = ['ct']
 _SCANS_WITH_MOTORS = ['ascan', 'dscan']
 _SPEC_SCAN_NAMES = _SCANS_WITH_MOTORS + _SCANS_WITHOUT_MOTORS
 _NOT_IMPLEMENTED_SCAN = 'Other'
-_BLUESKY_PLAN_NAMES = ['Scan', 'RelativeScan', 'Count']
+_BLUESKY_PLAN_NAMES = ['scan', 'relative_scan', 'count']
```

Comment:
@tacaswell is there a bluesky PR that you can link here so I know when these are renamed to ['ascan', 'dscan', 'ct']?